src/backend/joseki.be/webapp/Models/InfrastructureOverviewDiff.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+
+namespace webapp.Models
+{
+    /// <summary>
+    /// Returns infrastructure summary: scores by category, counters, etc.
+    /// </summary>
+    public class InfrastructureOverviewDiff
+    {
+        /// <summary>
+        /// First overall infrastructure summary.
+        /// </summary>
+        public InfrastructureComponentSummary Overall1 { get; set; }
```

Comment:
I guess, for `diff` view, you do not need `ScoreHistory` and `ScoreTrend`, that are part of `InfrastructureComponentSummary`. Maybe it makes sense to add another data-type for this?

Reply:
now we have `InfrastructureComponentSummary` and `InfrastructureComponentSummaryWithHistory` for this purpose.